### ISSUE TITLE
Updated files not reflected in the Android Client

### ISSUE THREAD

**Issue by maghog** [NewProggie/SparkleShare-Android#51]
If I have downloaded a file and it get an update I cant get the new version even if I do a refresh. The device seems to think its the same version for ever....

Magnus

**Comment by NewProggie** [comment]
Yes, terrible mistake.
Will be fixed in the next version coming soon.

**Comment by maghog** [comment]
Any hint when you expect to release version 1.1?

**Comment by NewProggie** [comment]
As part of fixing this bug I will implement a caching feature on the device as well, so that a non-changing directory will be cached on the device. This feature is very close related to this bug.
I hope I will be finished by the end of this week. A new version will then be released to the market.

**Comment by maghog** [comment]
Any news regarding release of v1.1??

**Comment by maghog** [comment]
Hi Kai,

I understand if you're busy but can you give some time frame about the 1.1 release?
Regards,
Magnus

**Comment by NewProggie** [comment]
Hi Magnus,

sorry for the delay.
Unfortunately, I wasn't able to work much on SparkleShare-Android the last few weeks, but I'm confident that the upcoming holidays will be a good time for me to work on the 1.1 release.
Thanks for you commitment anyways:)

Greetings,

Kai

Am 29.11.2011 um 08:47 schrieb maghog:

**Comment by maghog** [comment]
Thanx for the info :-)

**Comment by ghost** [comment]
May I suggest my fix in commit 229e1c1b44a9c671b2a42c2f785f5bcb17c3fa15 to remedy this issue. Of course, if you have your own way to do it, that is fine too.

**Comment by NewProggie** [comment]
@andrewjbarr Yes. This is a nice temp solution. Please open up a new pull request with this commit.
